tests/scripts/check_names.py

```diff
@@ -775,14 +812,15 @@ def check_symbols_declared_in_header(self):
         Perform a check that all detected symbols in the library object files
         are properly declared in headers.
         Assumes parse_names_in_source() was called before this.
-
```

Comment:
Various blank lines are removed in the backport, could you re-add these? The more similarity in this script between 2.28 and development, the more likely that future backports will apply cleanly.

Reply:
Done!